aibolit/model/model.py

```diff
@@ -54,7 +54,7 @@ def scale_dataset(
     metrics_codes_set = [x['code'] for x in config['metrics']]
     exclude_features = set(config['patterns_exclude']).union(set(config['metrics_exclude']))
     used_codes = set(features_conf['features_order'])
-    used_codes.add('M4')
+    used_codes |= {'M4', 'M6', 'M9', 'M10'}
     not_scaled_codes = set(patterns_codes_set).union(set(metrics_codes_set)).difference(used_codes).difference(
```

Comment:
can we just add a parameter, 1 parameter target. And then we will add
` used_codes.add(target)`

Reply:
Done.